### ISSUE TITLE
Unlogged player can move while logging timeout

### ISSUE THREAD

**Issue by Oshi41** [Chocorean/authmod#10]
Hello!
Unlogged player can move in logging timeout, but after reconnection he is standing in the new pos, not there where he was at the first time. On playerLogOutEvent you just need to spawn him at the first destination like this

```
@SubscribeEvent(priority= EventPriority.HIGHEST)
    public static void onLeave(PlayerLoggedOutEvent event){

           if (descriptors.contains(event.player)
               entity.setPositionAndUpdate(
                          dc.getPosition().getX(),
                          dc.getPosition().getY(),
                          dc.getPosition().getZ()
                  );
        logged.remove(event.player);
    }
```

**Comment by Chocorean** [comment]
Thanks for your feedback. This will be fixed soon!

Envoyé de mon iPhone

**Comment by Chocorean** [comment]
Fixed in release 2.5.
